cmd/prom-migrator/main.go

```diff
@@ -130,6 +131,8 @@ func parseFlags(conf *config, args []string) {
 		"Setting this value less than zero will indicate all data from mint upto now. ")
 	flag.StringVar(&conf.maxBlockSize, "max-read-size", "500MB", "(units: B, KB, MB, GB, TB, PB) the maximum size of data that should be read at a single time. "+
 		"More the read size, faster will be the migration but higher will be the memory usage. Example: 250MB.")
+	flag.IntVar(&conf.concurrentPulls, "concurrent-pulls", 1, "Concurrent pulls enables fetching of data parallelly. "+
+		"This enables higher throughput for significantly faster pulling of data remote-read storages.")
```

Comment:
```suggestion
		"May enable higher throughput by pulling data faster from remote-read stores.")
```

Reply:
Sorry, we are definite about this. Parallel fetching will always increase the throughput since requests sent are concurrent.

Reply:
If remote-read system is slow (like really slow), issuing more requests can lead to congestion